lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,46 @@
+module MigrationSyncHelper
+  def self.migrations_column_present?
+    ActiveRecord::Base.connection.columns("miq_regions").map(&:name).include?("migrations_ran")
```

Comment:
Would `detect` be better? `ActiveRecord::Base.connection.columns("miq_regions").detect { |column| column.name == "migrations_ran" }`